lms/resources/_js_config/__init__.py

```diff
@@ -538,3 +539,23 @@ def _sync_api(self):
             return self._blackboard_sync_api()
 
         return None
+
+    def _add_deeplinking_api(self):
+        if not self._context.is_canvas:
+            return
+
+        config = {
+            "data": {
+                "content_item_return_url": self._context.lti_params[
+                    "content_item_return_url"
+                ],
+            },
+        }
+
+        if self._application_instance.lti_version == "1.3.0":
+            config["path"] = self._request.route_path("lti.deeplinking.formfields")
+            config["data"]["deep_linking_settings"] = self._context.lti_params[
+                "deep_linking_settings"
+            ]
+
+            self._config["filePicker"]["deeplinkingAPI"] = config
```

Comment:
Is this a bug? Why doesn't this start with:

```
if not self._context.is_canvas or self._application_instance.lti_version != "1.3.0":
    return
```

Should this line be out-dented? Did the tests miss this?

Reply:
No, it's doing what's is supposed to do but it's organized weirdly as I split this from a PR that did this for both 1.1 and 1.3.

Moved the 1.3 conditional to the early return for this  PR :+1: